src/condor_ce_router_defaults

```diff
@@ -68,6 +68,17 @@ JOB_ROUTER_DEFAULTS = """JOB_ROUTER_DEFAULTS_GENERATED @=jrd
                                                   default_queue,
                                                   "")));
 
+    /* Request GPUs for whole node jobs (HTCONDOR-103) */
+    copy_RequestGPUs = "orig_RequestGPUs";
+    eval_set_OriginalGPUs = ifThenElse(orig_RequestGPUs isnt undefined, orig_RequestGPUs, undefined);
```

Comment:
You _might_ be able to simplify this to
```suggestion
    eval_set_OriginalGPUs = orig_RequestGPUs;
```
... along with a comment on why it's needed in the first place. Not certain though.

Reply:
In my testing this seems to work!